frontend/src/api/operatePreferenceData.test.ts

```diff
@@ -40,4 +33,19 @@ describe ('use of updateNickname method', () => {
       changeUserNotificationSpy.mockRestore()
     })
   })
+})
+
+describe ('test use of change avatar', () => {
+  it('test usage of upload new avatar method of OperateUserData class', () => {
+    const payload = new FormData()
+    payload.append('avatar', 'hello.png')
+    const avatarUploadSpy = jest.spyOn(operateUserDataRequest, 'uploadNewAvatar')
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    //@ts-ignore
```

Comment:
Please do not use `@ts-ignore` here. I'm almost sure that error here is because `uploadNewAvatar` expecting `{data: FormData}` as payload, but you povide just `FormData`